### PR TITLE
10-make-edit-member-form

### PR DESCRIPTION
**Pull request by kravocados**
button fix

### CODE REVIEW

**Review by kravocados — COMMENTED** [review]
nivii okay, so you have code for .button which is used across all pages so we need to change that name to something specific when you align it to flex end (line 69). 

we have a css file for the button information so you dont need to add alot of code present, just need to align the button to fit.

My advice is to place the button in a <div> and name this very specific. On your editMember.php file.

<div class="editMemberbutton">
<button class......> //  then have your button component nesting under. 

then in the .css place ur div class and have the alignment as flex-end.

Let me know if you need help

**Review by VaissheenaviPrabakaran — DISMISSED** [review]
whoops- okay, will look into it!! thankieww :))

File: styles/editMember.css [hunk 1]
```diff
@@ -96,3 +60,15 @@
 .buttonCancel[disabled]:hover {
   cursor: not-allowed;
 }
+
+.button{
+  color: var(--clr-white);
+  padding: 12px 0;
+  border-radius: 7px;
+  font-weight: 600;
+  align-self: flex-end;
```
Comment (kravocados) [inline styles/editMember.css]: main ! thing to fix

**Review by VaissheenaviPrabakaran — APPROVED** [review]
all good now :))

**Pull request merged**